src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search_form"
+      method="get"
+      data-qa="big"
+      action="/"
+    >
+      <label>
+        <input
+        class="search_field"
+        name="search"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
```

Comment:
Pay attention on indents
```suggestion
        <input
          class="search_field"
          name="search"
          type="text"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```